src/terms/xc.jl

```diff
@@ -83,36 +100,58 @@ end
         E *= term.scaling_factor
         potential .*= term.scaling_factor
     end
-    ops = [RealSpaceMultiplication(basis, kpt, potential[:, :, :, kpt.spin])
-           for kpt in basis.kpoints]
-    (E=E, ops=ops)
+    ops = map(basis.kpoints) do kpt
+        if haskey(terms, :vtau) && any(x -> abs(x) ≥ term.potential_threshold, terms.vtau)
+            # Need meta-GGA non-local operator
+            # Note: Minus in front of scaling coefficient comes from partial integration
+            Vτ = -term.scaling_factor .* permutedims(terms.vtau, (2, 3, 4, 1))
```

Comment:
pretty confusing to have Vτ be -vtau... are these names standard?

Reply:
Actually it's probably better to put the minus in the operator: `-div(Agrad)`. That way it's a positive operator when A is positive, and the standard kinetic energy when A=1/2.

Reply:
oh nothing is really standard in this regime. There is at least two philosophies in naming DFT things in the literature. `vtau` is what Libxc uses. I agree about the naming and the scaling, I'll put the constants into the operator. Then if V=1 it is just the kinetic energy (in my original implementation I forgot an 1/2).